scripts/build-single-manifests.sh

```diff
@@ -19,7 +19,6 @@ function generate_all_in_one_manifest()
 }
 
 
-generate_all_in_one_manifest 'config/base' 'deploy/single/all-in-one-dbless-legacy.yaml'
```

Comment:
Should this happen in this PR? 🤔 It seems to me it's not directly connected with requiring the version of the Gateway to be >= 3.4.1, is it? If we remove that, I think we should also put a changelog note for this.

Reply:
I did it because it is only used by `TestDeployAllInOneDBLESSLegacy`, which is removed in this PR. To get rid of all of them we have https://github.com/Kong/kubernetes-ingress-controller/issues/4712, I'm reverting this file. It will be more clear